packages/styles/settings-tools/_s.inline-icons.scss

```diff
@@ -26,4 +26,20 @@
   @if $icon == 'delete-24' {
     @return '<svg xmlns="http://www.w3.org/2000/svg" fill="#{$fill}" viewBox="0 0 24 24"><path d="M18.08 8a1 1 0 0 0-1.08.92L16.08 20H7.92L7 8.92a1 1 0 1 0-2 .16l1 12A1 1 0 0 0 7 22h10a1 1 0 0 0 1-.92l1-12A1 1 0 0 0 18.08 8z"/><path d="M19 5h-3.77l-.65-2.27a1 1 0 0 0-1-.73h-3.2a1 1 0 0 0-1 .73L8.77 5H5a1 1 0 0 0 0 2h14a1 1 0 0 0 0-2zm-8.62-2h3.24l.57 2H9.81zM12.5 18v-8a.5.5 0 0 0-1 0v8a.5.5 0 0 0 1 0zM14.25 18.5a.5.5 0 0 0 .5-.47l.5-8a.5.5 0 0 0-.47-.53.49.49 0 0 0-.53.47l-.5 8a.5.5 0 0 0 .47.53zM9.75 18.5a.5.5 0 0 0 .47-.53l-.5-8a.49.49 0 0 0-.53-.47.5.5 0 0 0-.47.53l.5 8a.5.5 0 0 0 .53.47z"/></svg>';
   }
+
+  @if $icon == 'notification-information-48' {
+    @return '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><g><rect fill="none"  width="48" height="48"/></g><g><path fill="#{$fill}" d="M24,32.75a1.5,1.5,0,0,1-1.5-1.5v-9a1.5,1.5,0,0,1,3,0v9A1.5,1.5,0,0,1,24,32.75Z"/><circle fill="#{$fill}" cx="24" cy="16.75" r="1.5"/><path fill="#{$fill}" d="M24,7A17,17,0,1,1,7,24,17,17,0,0,1,24,7m0-3A20,20,0,1,0,44,24,20,20,0,0,0,24,4Z"/></g></svg>';
+  }
```

Comment:
You have to clean up all svg, when you add one. you can delete this '<rect fill="none"  width="48" height="48"/>' and <g></g>. It's better to put the 'fill="#{$fill}"' inside the svg element as othe others svg.